sdk/eventhub/mock-hub/README.md

```diff
@@ -0,0 +1,48 @@
+## Mock Event Hubs service
+
+This package exposes a mock Event Hubs service for use in testing.
+
+Currently the mock service represents a single Event Hub within a namespace.
+
+### Goals
+
+This project was spun up for the following reasons:
+
+1. Avoid hitting the live service for tests.
+2. Deterministically trigger 'bad' service behavior.
+3. Simplify tests that rely on state. (populate events, change consumer groups/partition ids)
+4. Extend to support record & playback.
+
+Currently, the project only fulfills the 1st goal.
```

Comment:
💔

Reply:
Yes, but you can help me meet the other goals!